rst/backends.rst

```diff
@@ -374,6 +374,67 @@ The S3 compatible backend accepts the following backend options:
 .. __: https://doc.s3.amazonaws.com/proposals/copy.html
 
 
+Backblaze B2
+============
+
+Backblaze B2 is a cloud storage with its own API.
+
+The storage URL for backblaze b2 storage is ::
+
+   b2://<bucket-name>[/<prefix>]
+
+*bucket-name* is an (existing) bucket which has to be accessible with
+the provided account key. The *prefix* will be appended to all names
+used by S3QL and can be used to hold separate S3QL repositories in the
+same bucket.
+
+.. option:: account-id
+
+   Some API calls need the account id which can be passed by using
+   this parameter. If it is not provided, a separate call has to be
+   made to fetch it from the backend.
```

Comment:
Just realised, that this option is not needed (anymore?). The `accountId` gets returned from `b2_authorize_account` so no need for an extra API call.

Reply:
You are right. :slightly_smiling_face: